examples/closure-app/silabs/src/ClosureManager.cpp

```diff
@@ -124,26 +126,28 @@ void ClosureManager::CancelTimer()
 
 void ClosureManager::InitiateAction(AppEvent * event)
 {
-    Action_t action = static_cast<Action_t>(event->ClosureEvent.Action);
+    Action_t eventAction = static_cast<Action_t>(event->ClosureEvent.Action);
 
     ClosureManager & instance = ClosureManager::GetInstance();
 
+    // ideally, we should not receive an event for a different action while another action is ongoing.
+    // But due to asynchronous processing of command and stop command, this can happen if stop is received
```

Comment:
```suggestion
    // But due to asynchronous processing of commands and synchronous processing of the stop command, this can happen if stop is received
```